superset/assets/src/views/dashboardList/DashboardList.tsx

```diff
@@ -56,113 +68,120 @@ class DashboardList extends React.PureComponent<Props, State> {
-          },
-        }: any) => (
-            <span className='no-wrap'>{moment(changed_on).fromNow()}</span>
-          ),
-        Header: this.state.labelColumns.changed_on || '',
-        accessor: 'changed_on',
-        sortable: true,
-      },
-      {
-        Cell: ({ row: { state, original } }: any) => {
-          const handleDelete = () => this.handleDashboardDeleteConfirm(original);
-          const handleEdit = () => this.handleDashboardEdit(original);
-          if (!this.canEdit && !this.canDelete) {
-            return null;
-          }
-
-          return (
-            <span className={`actions ${state && state.hover ? '' : 'invisible'}`}>
-              {this.canDelete && (
-                <span
-                  role='button'
-                  className='action-button'
-                  onClick={handleDelete}
-                >
-                  <i className='fa fa-trash' />
-                </span>
-              )}
-              {this.canEdit && (
-                <span
-                  role='button'
-                  className='action-button'
-                  onClick={handleEdit}
-                >
-                  <i className='fa fa-pencil' />
-                </span>
-              )}
-            </span>
-          );
+  public columns = [
+    {
+      Cell: ({
+        row: {
+          original: { url, dashboard_title },
+        },
+      }: any) => <a href={url}>{dashboard_title}</a>,
+      Header: t('Title'),
+      accessor: 'dashboard_title',
+      filterable: true,
+      sortable: true,
+    },
+    {
+      Cell: ({
+        row: {
+          original: { changed_by_name, changed_by_url },
+        },
+      }: any) => <a href={changed_by_url}>{changed_by_name}</a>,
+      Header: t('Changed By Name'),
+      accessor: 'changed_by_fk',
+      sortable: true,
+    },
+    {
+      Cell: ({
+        row: {
+          original: { published },
+        },
+      }: any) => (
+          <span className='no-wrap'>{published ? <i className='fa fa-check' /> : ''}</span>
+        ),
+      Header: t('Published'),
+      accessor: 'published',
+      sortable: true,
+    },
+    {
+      Cell: ({
+        row: {
+          original: { changed_on },
         },
-        Header: 'Actions',
-        id: 'actions',
+      }: any) => (
+          <span className='no-wrap'>{moment(changed_on).fromNow()}</span>
+        ),
+      Header: t('Changed On'),
+      accessor: 'changed_on',
+      sortable: true,
+    },
+    {
+      Cell: ({ row: { state, original } }: any) => {
+        const handleEdit = () => this.handleDashboardEdit(original);
+        const handleExport = () => this.handleBulkDashboardExport([original]);
+        if (!this.canEdit && !this.canDelete && !this.canExport) {
+          return null;
+        }
+
+        return (
+          <ConfirmStatusChange title={t('Please Confirm')} description={`${t('Are you sure you want to delete')} ${original.dashboard_title}?`}>
```

Comment:
This `<ConfirmStatusChange>` could wrap just the delete button instead of the entire collection of buttons.